packages/@okta/vuepress-site/docs/reference/api/asa/projects/index.md

```diff
@@ -88,16 +88,16 @@ This endpoint returns a list of objects with the following fields and a `200` co
 | `create_server_users`   | boolean | (Optional) Whether to create Server Users for ASA Users in this Project. Defaults to `false`. If left `false`, the ASA User is responsible for ensuring that users that match the names of the Server Users in ASA exist on the server. |
 | `deleted_at`   | string | Time of deletion. `null` if not deleted. |
 | `force_shared_ssh_users`   | boolean | (Optional) If `true`, new Server Users will not be created for each ASA User in the Project. Instead they share a single standard user and a single admin user. Default is `false`. |
-| `forward_traffic`   | boolean | Whether to require that all traffic in the Project be forwarded through selected Gateways. Default is `false`. **Warning:** Requires a feature flag to be enabled. |
+| `forward_traffic`   | boolean | Whether to require that all traffic in the Project be forwarded through selected Gateways. Default is `false`. |
 | `id`   | string | The UUID of the Project |
 | `name`   | string | The name of the Project |
 | `next_unix_gid`   | number | (Optional) The GID to use when creating a new server user. |
 | `next_unix_uid`   | number | (Optional) The UID to use when creating a new server user. |
-| `rdp_session_recording`   | boolean | (Optional) Whether to enable remote desktop protocol (rdp) recording on all Servers in this Project. Default is `false`. **Warning:** Requires a feature flag to be enabled. |
+| `rdp_session_recording`   | boolean | (Not supported) Whether to enable remote desktop protocol (rdp) recording on all Servers in this Project. Default is `false`. |
```

Comment:
Thinking about it, since we intend to add RDP session recording by end of the year, it feels like maybe we can say something to that effect here. Maybe 'Reserved for future use, not supported'?